lib/pf/Authentication/Source/HtpasswdSource.pm

```diff
@@ -23,6 +23,7 @@ extends 'pf::Authentication::Source';
 has '+type' => (default => 'Htpasswd');
 has 'path' => (isa => 'Str', is => 'rw', required => 1);
 has 'stripped_user_name' => (isa => 'Str', is => 'rw', default => 'yes');
+has 'realm' => (isa => 'ArrayRef[Str]', is => 'rw');
```

Comment:
Since that field seems common to all sources, it should be moved in Authentication/Source.pm